src/Sentry/Laravel/Console/PublishCommand.php

```diff
@@ -151,4 +155,115 @@ private function askForDsnInput(): string
             }
         }
     }
+
+    private function askForJavaScriptSdk(string $dsn): void
+    {
+        if ($this->confirm('Do you want to install one of our JavaScript SDKs?', true)) {
+            $framework = $this->choice(
+                'Which frontend framework are you using?',
+                ['Vue.js', 'React', 'Angular', 'Svelte'],
+            );
+
+            $snippet = '';
+
+            switch ($framework) {
+                case 'Vue.js':
+                    $this->updateNodePackages(function ($packages) {
+                        return [
+                            '@sentry/vue' => '^7.36.0',
+                        ] + $packages;
+                    });
+                    $snippet = 'import * as Sentry from "@sentry/vue";' . PHP_EOL . PHP_EOL;
```

Comment:
m: Can we keep these snippets in js files we read from file system? This way we can also validate if syntax is valid (also makes diff's easier to read in the future).

Reply:
Sure can do, it's also less of a string concatenation mess 😬